hooks/usePWA.ts

```
@@ -99,16 +245,77 @@ export function usePWA(): UsePWAReturn {
+        (window as any).gtag('event', 'pwa_install_rejected', {
+          event_category: 'PWA',
+          event_label: 'User rejected installation',
+        });
+      }
     }
     
     setDeferredPrompt(null);
   };
 
   const applyUpdate = (): void => {
     if (registration && registration.waiting) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)